source/iface/context.rst

```diff
@@ -18,12 +18,9 @@ Contexts
 
   class context;
```

Comment:
I believe that we should wait a bit for updates on this [issue](https://github.com/KhronosGroup/SYCL_Reference/issues/72#issuecomment-1750665911), maybe it'll be needed to update the synopsis here